lib/storage.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+const Promise = require('bluebird');
+const createError = require('http-errors');
+const debug = require('debug')('pokeapp:storage');
+const fs = Promise.promisifyAll(require('fs'), {suffix: 'Prom'});
+
+const mkdirp = Promise.promisifyAll(require('mkdirp'));
+const storage = module.exports = {};
+
+storage.createItem = function(schemaName, item){
+  debug('createItem()');
+  if(schemaName) return Promise.reject(createError(400, 'expected schemaName'));
```

Comment:
Your POST tests are failing because of Line 13. Look at it closely to see what you're missing. Come see me if you can't figure it out.